spec/services/activity/updater_spec.rb

```diff
@@ -45,6 +45,46 @@
           )
         end
       end
+
+      ["programme", "project"].each do |type|
```

Comment:
Loops in specs 😱

Should the placeholder here be `level`?

Reply:
I couldn't remember if we liked them or not... I _think_ this one is fairly simple (no "sending" etc) unlike the importer spec which has some gnarly stuff going on, but let me know if you'd like me to get rid of the loops. Will update the placeholder.

Reply:
I just remember some difficulty debugging stuff when I was going hard on looping before, but I like the DRYness. It's not a super complicated block, as you say, so I think it's fine